app/scripts/models/sync-engines.js

```diff
@@ -60,7 +60,7 @@ define((require, exports, module) => {
       text: t('Preferences')
     },
     {
-      checked: false,
+      checked: true,
       id: 'addresses',
       // We know addresses will be available in Firefox 56, even before
```

Comment:
drive-by - I'd consider removing this soon-to-be baggage now and treating it just like CCs